hydra-tx/test/Hydra/Tx/Contract/Deposit.hs

```diff
@@ -21,4 +29,24 @@ healthyDepositTx =
       depositDeadline
 
 healthyDepositUTxO :: UTxO
-healthyDepositUTxO = genUTxOAdaOnlyOfSize 1 `generateWith` 42
+healthyDepositUTxO = resize 3 genUTxO `generateWith` 42
```

Comment:
Not sure since when, but `genUTxO`, `genTxOut`, etc generate HUGE quantities in `Value` and the `cardano-ledger` regularly chokes on putting those into transaction outputs (through `error` calls in partial functions :vomiting_face:). `resize 3` here does not create particularly small `Value`, but just luckily a small enough one.

We should test and fix the generators..